modules/test/playwright/tests/portlet-configuration-web/rolePermissions.spec.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+// @ts-ignore
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {loginTest} from '../../fixtures/loginTest';
+import {portletConfigurationPermissionsPageTest} from '../../fixtures/portletConfigurationPermissionsPagesTest';
+
+export const test = mergeTests(
+	loginTest(),
+    portletConfigurationPermissionsPageTest
+);
+
+
+test ('LPD-25265 search results should stay when form submitted', async ({
+    page,
+    portletConfigurationPermissionsPage,
+}) => {
+    await portletConfigurationPermissionsPage.goto();
+    await expect(
+        portletConfigurationPermissionsPage.searchBar
+    ).toBeVisible();
+    await page.waitForTimeout(500);
```

Comment:
why is this needed?

Reply:
I was experiencing some issues with the modal loading occasionally taking a really long time and the test failing due to it. I'll remove if you prefer.